enhttp_test.go

```diff
@@ -20,6 +20,7 @@ const (
 func TestRoundTrip(t *testing.T) {
 	_, counter, err := fdcount.Matching("TCP")
 	defer func() {
+		time.Sleep(1 * time.Second)
```

Comment:
Is it worth adding a test condition to count goroutines?

Reply:
Nice idea. Added